action_name:
  - move_kinect_action
move_kinect_action:
  pitch_step: 0.0
  min_pitch: -0.785
  max_pitch: 0.785
  offset_pitch: -0.05
  yaw_step: 0.7
  min_yaw: -1.308
  max_yaw: 1.308
  offset_yaw: 0
  scan_rate: 1.0
  pitch_rate: 0.1
  command_timeout: 10.0
  movement_threshold: 0.05
  lax_movement_threshold: 0.5
  imu_topic: /sensors/imu_rpy
  buffer_size: 5
  pitch_command_topic: /kinect_pitch_controller/command
  yaw_command_topic: /kinect_yaw_controller/command
  pitch_joint_parent: base_link
  pitch_joint_child: kinect_pitch_servo_link
  yaw_joint_parent: kinect_yaw_servo_link
  yaw_joint_child: kinect_platform_link
  sensor_frame: kinect_frame
